optools/run_bvt.sh

```diff
@@ -23,7 +23,7 @@ PROXY=${3:-}
 
 function launch_mo() {
     cd $MO_WORKSPACE
-    mo_reuse_enable_checker_verbose=true ./mo-service -debug-http=:12345 -launch ./etc/$LAUNCH/launch.toml $PROXY &>mo-service.log &
+    mo_reuse_enable_checker=true mo_reuse_enable_checker_verbose=true ./mo-service -debug-http=:12345 -launch ./etc/$LAUNCH/launch.toml $PROXY &>mo-service.log &
```

Comment:
you cannot update this file. mo_reuse_enable_checker_verbose=true already enable check